.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
Make sure you don't publish a release if it's a pull request, this can be done using `if: github.event_name == 'push' && github.ref == 'refs/heads/master` in the steps that you want to skip.

Reply:
Should I just remove the `pull_request:` trigger?

Reply:
I think it's useful. It's nice to know that the build will still pass after the PR is merged.